autorag/evaluate/metric/retrieval_contents.py

```diff
@@ -15,7 +15,13 @@
 def retrieval_contents_metric(func):
     @functools.wraps(func)
     def wrapper(gt_contents: List[List[str]], pred_contents: List[List[str]]) -> List[float]:
-        return list(map(lambda x: func(x[0], x[1]), zip(gt_contents, pred_contents)))
+        results = []
+        for gt, pred in zip(gt_contents, pred_contents):
+            if gt == []:
```

Comment:
The fetched contents can be [''] => empty string or [' '] => empty whitespace
Also pred_contents, too.
So, it will be great you add `any(bool(g) is False for g in gt)` condition.